container/go/pkg/utils/utils.go

```diff
@@ -44,3 +49,49 @@ func ReadImage(src, format string) (v1.Image, error) {
 
 	return nil, errors.Errorf("unknown image format %q", format)
 }
+
+// TODO: REMOVE these two functions copied from Winnie's PR, refactor after her PR is merged.
```

Comment:
State which two functions are being referred to here and link the PR instead of saying `Winnie's PR`.